models/wallets.js

```diff
@@ -0,0 +1,77 @@
+const firestore = require('../utils/firestore')
+const walletModel = firestore.collection('wallet')
+
+/**
+ * Fetches the data from user wallet
+ * @return {Promise<walletModel|object>}
+ */
+const fetchWallet = async (userId) => {
+  try {
+    const walletDocs = await walletModel.where('userId', '==', userId).limit(1).get()
+    const userWallet = walletDocs.docs[0]
+    if ((!walletDocs.empty) && userWallet) {
+      return { id: userWallet.id, ...userWallet.data() }
+    }
+    return {}
+  } catch (err) {
+    logger.error('Error retrieving wallets', err)
+    return err
+  }
+}
+
+/**
+ * Create new wallet for user
+ * @return {Promise<walletModel|object>}
+ */
+const createWallet = async (userId) => {
+  try {
+    const walletData = {
+      userId,
+      isActive: true,
+      currency: {}
+    }
+    const { id } = await walletModel.add(walletData)
```

Comment:
user-wallet is a one-one relation
so keeping the userId == walletId would be easier right?